Splunk - handle notable raw fields where key equals value (#10353)
demisto/content: Packs/SplunkPy/ReleaseNotes/1_2_6.md

#### Integrations
##### SplunkPy

> - Fixed an issue where a notable event raw field that was provided wrapped in double quotes was returned with the key as the value in the incident metadata if it did not exist in the notable event.

Reviewer: ```suggestion
- Fixed an issue where providing a raw field which is wrapped in double quotes in a notable event, would return with the key set as the value in the incident metadata if it did not exist.
```
